config/client_config/me/system/config/templates/features/aca_individual_market/eligibilities.yml

```diff
@@ -126,6 +127,13 @@ registry:
             item: 'gid://enroll_app/Family::FamilyMember'
           - key: :evidence_ref
             item: 'gid://enroll_app/Person'
+      - key: :me_residency
```

Comment:
similar to above, we should not be using a named client/state in a key or property name